servers/visual/visual_server_scene.cpp

```diff
@@ -2138,14 +2136,19 @@ void VisualServerScene::_bake_gi_probe_light(const GIProbeDataHeader *header, co
 					success_count++;
 				}
 			}
-			//print_line("BAKE TIME: " + rtos((OS::get_singleton()->get_ticks_usec() - us) / 1000000.0));
-			//print_line("valid cells: " + itos(success_count));
+
+#ifdef DEBUG_ENABLED
+			print_line("BAKE TIME: " + rtos((OS::get_singleton()->get_ticks_usec() - us) / 1000000.0));
```

Comment:
To be honest I'm not sure if we want to be spammed by this every time using GI probes with a debug build of the editor (which is the default when building from source).

This should either be shown only in verbose more, or more likely commented out again or removed as it's only meant as debugging info for whoever is hacking that code.

Reply:
I'm happy to move it into a more verbose mode, it just seemed like it may be useful for debugging slowness in the actual project. Maybe when using probes in a scene too busy for the hardware.

My thinking is that if these lines are useful for debugging the engine they should just be there rather than require someone to uncomment them. People less familiar with the engine won't know they are there.

Reply:
I mean they are relevant for developers debugging this class, not engine users. And someone who is editing the file already in the IDE would notice the comments and could uncomment them if they want to see the debug info :)

Reply:
I guess my point is more: how would you even know to look there if you're trying to debug the problems that these print lines were put in for to help solve.

Reply:
Well you would have an issue with GI Probes, and would check the relevant code to try to debug it, seeing that there are some debug statements just waiting for you to check their return value. Those statements are not meant for normal users who find their light baking is too slow. They will just make a bug report saying "it's slow", and devs will check and debug it.

Those print_line were added by @reduz as he developed the feature, to be able to see what happens. He's done developing it, and kept them commented out for next time he has to dive it in. Either we get rid of them, or keep them commented out.

Reply:
OK, if that's your stance I'll comment them back out, including the timer variable this time.